components/resources/commands.grdp

```diff
@@ -0,0 +1,472 @@
+  </message>
+  <message name="IDS_IDC_SELECT_PREVIOUS_TAB">
+    Previous tab
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_0">
+    Select tab 0
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_1">
+    Select tab 1
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_2">
+    Select tab 2
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_3">
+    Select tab 3
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_4">
+    Select tab 4
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_5">
+    Select tab 5
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_6">
+    Select tab 6
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_7">
+    Select tab 7
+  </message>
+  <message name="IDS_IDC_SELECT_LAST_TAB">
+    Select last tab
+  </message>
+  <message name="IDS_IDC_DUPLICATE_TAB">
+    Duplicate tab
+  </message>
+  <message name="IDS_IDC_MOVE_TAB_NEXT">
+    Move tab next
+  </message>
+  <message name="IDS_IDC_MOVE_TAB_PREVIOUS">
+    Move tab previous
+  </message>
+  <message name="IDS_IDC_SEARCH">
+    Search
+  </message>
+  <message name="IDS_IDC_WINDOW_MENU">
+    Window menu
+  </message>
+  <message name="IDS_IDC_MINIMIZE_WINDOW">
+    Minimize window
+  </message>
+  <message name="IDS_IDC_MAXIMIZE_WINDOW">
+    Maximize window
+  </message>
+  <message name="IDS_IDC_ALL_WINDOWS_FRONT">
+    Show all windows
+  </message>
+  <message name="IDS_IDC_USE_SYSTEM_TITLE_BAR">
+    Toggle system title bar
+  </message>
+  <message name="IDS_IDC_OPEN_IN_PWA_WINDOW">
+    Open in PWA window
+  </message>
+  <message name="IDS_IDC_WEB_APP_SETTINGS">
+    Web app settings
+  </message>
+  <message name="IDS_IDC_WEB_APP_MENU_APP_INFO">
+    Web app info
+  </message>
+  <message name="IDS_IDC_BOOKMARK_THIS_TAB">
+    Bookmark tab
+  </message>
+  <message name="IDS_IDC_BOOKMARK_ALL_TABS">
+    Bookmark all tabs
+  </message>
+  <message name="IDS_IDC_EMAIL_PAGE_LOCATION">
+    Email page URL
+  </message>
+  <message name="IDS_IDC_BASIC_PRINT">
+    Basic print
+  </message>
+  <message name="IDS_IDC_SAVE_CREDIT_CARD_FOR_PAGE">
+    Save credit card for page
+  </message>
+  <message name="IDS_IDC_TRANSLATE_PAGE">
+    Translate page
+  </message>
+  <message name="IDS_IDC_ROUTE_MEDIA">
+    Route media
+  </message>
+  <message name="IDS_IDC_WINDOW_MUTE_SITE">
+    Mute site
+  </message>
+  <message name="IDS_IDC_WINDOW_PIN_TAB">
+    Pin tab
+  </message>
+  <message name="IDS_IDC_WINDOW_GROUP_TAB">
+    Group tab
+  </message>
+  <message name="IDS_IDC_MIGRATE_LOCAL_CREDIT_CARD_FOR_PAGE">
+    Migrate local credit card for page
+  </message>
+  <message name="IDS_IDC_FOCUS_THIS_TAB">
+    Focus tab
+  </message>
+  <message name="IDS_IDC_QRCODE_GENERATOR">
+    QR code generator
+  </message>
+  <message name="IDS_IDC_SAVE_AUTOFILL_ADDRESS">
+    Save autofill address
+  </message>
+  <message name="IDS_IDC_OFFERS_AND_REWARDS_FOR_PAGE">
+    Offers and rewards for page
+  </message>
+  <message name="IDS_IDC_WEBAUTHN">
+    Web authn
```

Comment:
Should this be `authn`?

Reply:
Yup, but it probably should be one word instead of two